docs/settings/monitoring-settings.asciidoc

```diff
@@ -88,7 +88,7 @@ which has a default value of `30000`.
 `monitoring.ui.elasticsearch.ssl`::
 Shares the same configuration as <<elasticsearch-ssl-cert-key, `elasticsearch.ssl`>>. These settings configure encrypted communication between {kib} and the monitoring cluster.
 
-`monitoring.cluster_alerts.allowedSpaces` {ess-icon}::
+|[[kibana-monitoring-clusterAlerts-allowedSpaces]] `monitoring.cluster_alerts.allowedSpaces` {ess-icon}::
```

Comment:
```suggestion
[[kibana-monitoring-clusterAlerts-allowedSpaces]] `monitoring.cluster_alerts.allowedSpaces` {ess-icon}::
```

Reply:
I *think* this will fix the docs build error

Reply:
Thanks!